src/Fable.Cli/Main.fs

```diff
@@ -1140,31 +1141,49 @@ let private compilationCycle (state: State) (changes: ISet<string>) =
                 }
 
             // Sometimes errors are duplicated
-            let logs = Array.distinct logs
+            let logs = logs |> Array.distinct
```

Comment:
:)

Reply:
🙈

This is a left over of my initial implementation, I have it reverted locally